### PR TITLE
add Pioupiou/RibsAdminBundle

### PR DESCRIPTION
**Pull request by Piou-piou**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by Nyholm** [comment]
Thank you for this PR. Can you look why travis is unhappy?

**Comment by Piou-piou** [comment]
@Nyholm I will look those bugs later in the day

File: piou-piou/ribs-admin-bundle/0.1/manifest.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "Ribs\\RibsAdminBundle\\": ["all"]
```
Comment (stof) [inline piou-piou/ribs-admin-bundle/0.1/manifest.json]: this is invalid. the bundle class name does not end with a `\`

Comment (Piou-piou) [reply piou-piou/ribs-admin-bundle/0.1/manifest.json]: thanks i will fix that. I made some updates on package before to push that change

File: piou-piou/ribs-admin-bundle/0.1/manifest.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "PiouPiou\\RibsAdminBundle\\RibsAdminBundle": ["all"]
```
Comment (Nyholm) [inline piou-piou/ribs-admin-bundle/0.1/manifest.json]: Isnt you bundle called: `Ribs\\RibsAdminBundle\\RibsAdminBundle`?

Comment (Piou-piou) [reply piou-piou/ribs-admin-bundle/0.1/manifest.json]: Yes but it's an error that I have done before. I have installed a recipe server. I test all things on my server and i will push it on this merge later :)

File: piou-piou/ribs-admin-bundle/0.1/manifest.json (hunk 1 above)
Comment (Piou-piou) [reply piou-piou/ribs-admin-bundle/0.1/manifest.json]: all is fixed in last commit, you can close you review :)

File: piou-piou/ribs-admin-bundle/0.1/manifest.json (hunk 2 above)
Comment (Piou-piou) [reply piou-piou/ribs-admin-bundle/0.1/manifest.json]: all is fixed in last commit, you can close you review :)

File: piou-piou/ribs-admin-bundle/0.1/config/packages/ribs_admin.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                check_path: login_check
+                default_target_path: ribsadmin_index
+
+    access_control:
+        # require ROLE_ADMIN for /admin*
+        - { path: ^/ribs-admin/login$, role: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/ribs-admin/login, role: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/ribs-admin/register$, role: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/ribs-admin/register, role: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/ribs-admin, roles: ROLE_ADMIN }
```
[the remark this answers is not in the document]
Comment (Piou-piou) [reply piou-piou/ribs-admin-bundle/0.1/config/packages/ribs_admin.yaml]: add

**Pull request closed without merging**